forge-gui/res/cardsfolder/upcoming/sigarda_font_of_blessings.txt

```diff
@@ -0,0 +1,10 @@
+Name:Sigarda, Font of Blessings
+ManaCost:2 G W
+Types:Legendary Creature Angel
+PT:4/4
+K:Flying
+S:Mode$ Continuous | Affected$ Permanent.Other+YouCtrl | AddKeyword$ Hexproof | Description$ Other permanents you control have hexproof. 
+S:Mode$ Continuous | Affected$ Card.TopLibrary+YouCtrl | AffectedZone$ Library | MayLookAt$ You | Description$ You may look at the top card of your library any time.
+S:Mode$ Continuous | Affected$ Angel.TopLibrary+YouCtrl,Human.TopLibrary+YouCtrl | AffectedZone$ Library | MayPlay$ True | Description$ You may cast Angel spells and Human spells from the top of your library.
```

Comment:
Check if you can play back of _Blackbloom Rogue_ by it
then you need `+nonLand`

Reply:
I was only able to cast the front side

Reply:
It's probably needed otherwise you can play _Dryad Arbor_ when you have _Maskwood Nexus_

@Northmoc 
friendly ping for you to be aware of this corner case